board/stm32fx/stm32fx_config.h

```diff
@@ -2,11 +2,15 @@
   #include "stm32fx/inc/stm32f4xx.h"
   #include "stm32fx/inc/stm32f4xx_hal_gpio_ex.h"
   #define MCU_IDCODE 0x463U
+  #define BCDDEVICE 0x23
```

Comment:
generally prefer typed constants instead of defines, i.e. `const uint8_t` in this case